v2/Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: test
 test:
-	go test -race ./...
+	go test -race -count=1 ./...
```

Comment:
we do not need count=1 here

v2 is updated regularly, so we could run tests in parallel
execution is rarely updated, so it is required there, because otherwise test results will be cached, because the execution directory is not changed, but we are running the execution tests on each v2 update

Reply:
Right, thanks for explaining this!